pkg/action/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package action contains functions for creating actions to send to vehicles.
+package action
```

Comment:
Note: if kept, we could consider having `action` live under `vehicle` (so `pkg/vehicle/action`) (an idea).

Reply:
...or `internal/action`? What's the value in exporting? Plus, assigning generic names like "action" to exported packages is frowned on.